.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 3
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 30
+# Pull requests with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: false
```

Comment:
I wonder if sendign a comment helps in this case? As if its stale, there is 27 days limit so a user should know?

"The pull request has not been updated for 3 days, marking as stale. In case no update within the next 27 days, we will close it."